ghbackup/fetch.go

```diff
@@ -126,11 +126,17 @@ func getNextURL(header http.Header) string {
 	if len(parts) == 0 {
 		return ""
 	}
-	firstLink := parts[0]
-	if !strings.Contains(firstLink, "rel=\"next\"") {
+	var nextLink string
+	for _, v := range parts {
+		if strings.Contains(v, "rel=\"next\"") {
+			nextLink = strings.TrimSpace(v)
```

Comment:
**Question [clarification]:** 

Do we want to break here once we find a link starting containing `"rel=\"next\"" ` or is the last link with the string matching we're interested in?

Reply:
Basically, I wonder if there can there be a bunch of links containing the string `"rel=\"next\""`?

Reply:
Yeah there should be just one "next". Added a break.

Reply:
Am gonna merge this now.